### ISSUE TITLE
imread automatically converts colour to grey image

### ISSUE THREAD

**Issue by christian-rauch** [scikit-image/scikit-image#2817]
## Description
When loading some colour images with `imread`, the image is converted to grey-level without requesting the conversion.

## Way to reproduce
Example image `example.png` (it is mostly grey but contains a green region on the bottom):
![example](https://user-images.githubusercontent.com/8226248/31046051-d15c658a-a5e9-11e7-80a5-d6e92a4fcedf.png)

Code:
```
#!/usr/bin/env python
import skimage.io as io
import cv2

img = io.imread("example.png")
print "skimage",img.shape

cvimg = cv2.imread("example.png")
print "opencv",cvimg.shape
```

On my system I get:
```
skimage (240, 320)
opencv (240, 320, 3)
```
and the green colour (0,255,0) gets converted with skimage to grey 149.

I would expect from `io.imread` to return a 3-channel image, like it is provided by opencv's `imread`.

- scikit-image 0.13.1 (pip)
- Python 2.7.6
- Ubuntu 14.04

**Comment by stefanv** [comment]
I see the same (workaround: `io.imread(..., plugin='matplotlib')`.  We use PIL by default, so will have to dig into what goes wrong with that loader.

- This seems to be a paletted image.
- The function `_palette_is_greyscale` returns True, whereas it should return False.
- It looks like the palette is sliced incorrectly for the check, based on the extrema.

PR filed to fix that: https://github.com/scikit-image/scikit-image/pull/2818

**Comment by ahojnnes** [comment]
Fixed in #2818, thanks for reporting.
